README.md

```diff
@@ -1,2 +1,43 @@
+|---|---|---|---|---|
+| **When configured in Adobe Experience Platform Launch, ...**  | When configured in the Data Collection UI... | When referring to something "in" Launch, it is assumed that the author is referring to the UI.  | **When configured in Adobe Experience Platform Launch**, this extension sends AEM Asset Impression and Click metrics to Adobe Analytics.  | **When configured in the Data Collection UI**, this extension sends AEM Asset Impression and Click metrics to Adobe Analytics.  |
+
+The table below offers more comprehensive guidance on the varying uses of the original terms and how to apply the branding updates in a given situation. As any given term can be used in a variety of contexts, the table offers guidance on the nuance of the branding changes, including how they are applied and an explanation as to why.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+| **Platform Launch**  | Platform  | The reference to Launch is not always necessary if the sentence was referring to the service provided. In this case Launch can be omitted as tags is a set of capabilities and not a service. Here, Platform provides the functionality.  | **Platform Launch** passes `settings` into the rule’s condition-type library modules where conditions are then evaluated. | **Platform** passes `settings` into the rule’s condition-type library modules where conditions are then evaluated. |
+| **Platform Launch** | {ommitted} | If context has already been established, naming the capability in the sentence may be obsolete and can be removed entirely. | If the extensions, tags, and systems you are going to deploy through **Platform Launch** are very similar across your sites or applications, you might want to include them in the same property. | If the extensions, tags, and systems you are going to deploy are very similar across your sites or applications, you might want to include them in the same property. |
+| **An Adobe Experience Platform Launch {noun}** | a tags-enabled {noun} | When tags is providing the functionality to the noun, the hyphenated 'enabled' adds clarity to the sentence.  | This extension provides the functionality for adding the 'Media' tracker instance to **an Adobe Experience Platform Launch site** or project.  | This extension provides the functionality for adding the `Media` tracker instance to **a tags-enabled site** or project.   |
+| **The Launch runtime**  |  The tag runtime | Use singular version of Tag and "The" as an article when talking about a general concept/process rather than individual runtime events.  | **The Adobe Experience Platform Launch runtime** (delivered to clients' websites) is designed to support the following browsers:  | ****The tag runtime**** in Adobe Experience Platform (delivered to clients' websites) is compatible with multiple browsers:  |
```

Comment:
Wasn't it the "tags runtime"? 

Why is "Tag" capitalized, and not quoted? Why is "The" capitalized?

What do you mean by "about a general concept/process"...?